OpenRA.Mods.Common/Traits/ProximityCapturable.cs

```diff
@@ -189,7 +189,7 @@ void ChangeOwnership(Actor self, Actor captor)
 
 		void INotifyOwnerChanged.OnOwnerChanged(Actor self, Player oldOwner, Player newOwner)
 		{
-			skipTriggerUpdate = false;
+			Game.RunAfterTick(() => skipTriggerUpdate = false);
```

Comment:
Can you change the `self.Owner` in line 186 to `captor.Owner` while you're at it, please? The owner only changes after another frame end task queued by the actor itself, not during the one queued here, so the wrong player ends up getting the notification. Thanks!